Strategy11/formidable-forms
# Fix: Resolve Overlapping Issue After Deleting a Section in Form Builder

This PR addresses the overlapping problem that occurs in the form builder after a section with fields is deleted.

## Related Issue:
[Funny overlapping in form builder after deleting a section](https://github.com/Strategy11/formidable-pro/issues/3259)

## Depends on:
[Fix: Display "Add Fields Here" Message After Deleting All Fields](https://github.com/Strategy11/formidable-forms/pull/1348)

## QA URL
https://qa.formidableforms.com/sherv11/wp-admin/admin.php?page=formidable&frm_action=edit&id=1

## Testing Instructions:
1. Navigate to the form builder
2. Add a text field to a section
3. Save to get the update message
4. Delete the section
5. Verify that no funny overlapping occurs and the layout looks correct

## Output Comparison:
### Before
![2023-10-23 10 47 12](https://github.com/Strategy11/formidable-forms/assets/69119241/b326114a-2623-4401-8adc-7941d6e371ef)

### After
![2023-10-23 10 52 25](https://github.com/Strategy11/formidable-forms/assets/69119241/aa951fb5-8e52-400a-a129-489c5af1ac82)
